.github/workflows/deploy-helm-production.yaml

```diff
@@ -63,18 +87,13 @@ jobs:
 
   releaser:
     runs-on: ubuntu-latest
+    needs: [build, package-and-publish-helmchart]
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-      - name: Set up Go
-        uses: actions/setup-go@v5
+      - name: Download artifact
+        uses: actions/download-artifact@v4
         with:
-          go-version-file: go.mod
-      - name: Build linux-amd64
-        run: |
-          make build
+          name: manager
```

Comment:
hmm, thinking about this, the binary in the image will be CVE free, but this is the one that will end up in production and used by kots. i worked around a similar situation with the schemahero binary and the kotsadm-migrations image in kots by extracting the binary from the image published by apko: https://github.com/replicatedhq/kots/blob/main/.github/actions/build-custom-melange-package/action.yml#L44-L45

Reply:
changed